local_settings.py

```diff
@@ -42,3 +42,5 @@
 #DATABASE_PORT = '3306'
 
 SECRET_KEY = 'mydummysecret'
+
+CARBONLINK_HOSTS = ['127.0.0.1:10012', '127.0.0.1:10022', '127.0.0.1:10032', '127.0.0.1:10042', '127.0.0.1:10052', '127.0.0.1:10062', '127.0.0.1:10072', '127.0.0.1:10082']
```

Comment:
nit:
```
X = [
  'val1',
  ...
  'valN',
]
```
is usually better for readability/diffs imo.

Reply:
👍